packages/apps/client/src/stores/TriggerStore.ts

```diff
@@ -0,0 +1,77 @@
+import { observable, action } from 'mobx'
+import { EventEmitter } from 'eventemitter3'
+
+import { APIConnection, RootAppStore } from './RootAppStore'
+import { hardCodedTriggers } from '../lib/triggers/triggers'
+
+import { TriggerHandlerEvents } from '../lib/triggers/triggerHandlers/TriggerHandler'
+import { TriggerHandlerXKeys } from '../lib/triggers/triggerHandlers/TriggerHandlerXKeys'
+import { TriggerHandlerKeyboard } from '../lib/triggers/triggerHandlers/TriggerHandlerKeyboard'
+import { TriggerHandlerStreamdeck } from '../lib/triggers/triggerHandlers/TriggerHandlerStreamdeck'
+import { TriggerHandlerSpaceMouse } from '../lib/triggers/triggerHandlers/TriggerHandlerSpaceMouse'
+
+export interface TriggerStoreEvents {
+	action: TriggerHandlerEvents['action']
+}
+/**
+ * The TriggerStore is responsible for listening to triggers (eg keyboard shortcuts) and dispatching action events
+ */
+export class TriggerStore extends EventEmitter<TriggerStoreEvents> {
+	/** When set, indicates that a TriggerHandler wants to request access to HIDDevices */
+	public hidDeviceAccessRequests = observable.map<
```

Comment:
Should this be called `hidDeviceAccessRequests` or should it just be deviceAccessRequests? I suppose some of these might be MIDI or something?